lib/driver.js

```diff
@@ -817,8 +824,24 @@ class XCUITestDriver extends BaseDriver {
             // Stop the existing process before starting a new one to start a fresh WDA process every session.
             await this.mobileKillApp(this.wda.bundleIdForXctest);
 
-            // TODO: Define `prebuiltWDAPath` or something as a new capability.
-            // Then, we can start installing the WDA package to start a session without xcodebuild.
+            if (this.opts.prebuiltWDAPath) {
+              const candidateBundleId = await extractBundleId.bind(this)(this.opts.prebuiltWDAPath);
+              this.wda.updatedWDABundleId = candidateBundleId.replace('.xctrunner', '');
+              this.log.info(`Installing prebuilt WDA ${this.opts.prebuiltWDAPath}`);
+              // Note: The CFBundleVersion in the test bundle was always 1.
+              // It may not be able to compare with the installed versio.
+              await installToRealDevice(
+                // @ts-expect-error - do not assign arbitrary properties to `this.opts`
+                this.opts.device,
+                this.opts.prebuiltWDAPath,
+                candidateBundleId,
+                {
+                  skipUninstall: true,
```

Comment:
does it properly reinstall changed packages? maybe it would still be safer to uninstall it first?